src/core/exec-invoke.c

```diff
@@ -3033,13 +3033,35 @@ static int pick_versions(
         return 0;
 }
 
+static bool can_apply_cgroup_namespace(const ExecContext *context, const ExecParameters *params) {
+        int r;
+
+        r = cg_all_unified();
+        if (r < 0) {
+                log_exec_warning_errno(context, params, r, "Failed to determine cgroup hierarchy version: %m");
+                return false;
+        } else if (r == 0) {
```

Comment:
Not a big matter, but we usually drop unnecessary `else` like this, as the above branch has `return`.